### PR TITLE
chore: remove non needed tools

### PR DESCRIPTION
**Pull request by frezbo**
Remove swtpm and it dependencies (reverts commit 88ebb40dd348b6c9e4dc5551b616e4a1892b4e42)
Remove sd-boot (partially revert 271c4a66b6987d9de2c0d1d69891b5ff277ebd43)

https://github.com/siderolabs/talos/pull/7278 re-wrote sd-measure in Go and we don't need these tools anymore.

### CODE REVIEW

**Comment by smira** [comment]
I guess we still need `sd-boot` ?

**Comment by frezbo** [comment]
> I guess we still need `sd-boot` ?

yes

File: tools/pkg.yaml [hunk 1]
```diff
@@ -70,14 +67,11 @@ dependencies:
...
   - stage: sbsign
-  - stage: sd-boot
-  - stage: sd-measure
-  - stage: sd-stub
   - stage: sed
-  - stage: socat
   - stage: squashfs-tools
   - stage: swig
-  - stage: swtpm
+  - stage: sd-boot
```
Comment (smira) [inline tools/pkg.yaml:73]: nit: not-sorted :)

**Review by smira — APPROVED** [review]

**Pull request merged**